qml/pages/CameraUI.qml

```diff
@@ -9,7 +9,7 @@ Page {
     id: page
 
```

Comment:
The way it is now, the Silica "Glass" background flashes for a short time when rotating the interface. Ideally you could put the VideoOutput in ApplicationWindow instead of Page.
A quick workaround may be to leave almost everything as is and just not Animate the rotation, for example like this:
```suggestion
    defaultOrientationTransition: Transition {
        NumberAnimation {}
    }
```

Reply:
That works quite nicely